src/lstate.js

```diff
@@ -69,14 +69,23 @@ class global_State {
 
     constructor(L) {
         this.mainthread = L;
-        this.strt = null; // TODO: string hash table
+        this.strt = new Map();
```

Comment:
It will yes. I wrote a really simple and naive to begin with. I'm open to suggestions.